webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js

```diff
@@ -102,8 +102,8 @@ const ContentViewHistories = ({ cvId }) => {
             user,
           } = history;
           return (
-            <Tr key={`${versionId}_${createdAt}`}>
-              <Td key={createdAt}><LongDateTime date={createdAt} showRelativeTimeTooltip /></Td>
+            <Tr key={`${versionId}_${createdAt}_${index}`}>
+              <Td key={`${index}`}><LongDateTime date={createdAt} showRelativeTimeTooltip /></Td>
```

Comment:
the key on the first element of the parent shouldn't be needed here, also: 
{`${index}`} === {index}